### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by YuLiu-web**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ by Tingying Peng, Kurt Thorn, Timm Schroeder, Lichao Wang, Fabian J Theis, Carst
...
 
-You can also find examples of running the package at [folder Notebooks](https://github.com/peng-lab/PyBaSiC/tree/main/Notebooks). Data used in the examples and a description can be downloaded from [Dropbox](https://www.dropbox.com/s/plznvzdjglrse3h/Demoexamples.zip?dl=0)
+You can also find examples of running the package at [folder Notebooks](https://github.com/peng-lab/PyBaSiC/tree/main/Notebooks). Data used in the examples and a description can be downloaded from [Dropbox](https://www.dropbox.com/s/plznvzdjglrse3h/Demoexamples.zip?dl=0) or [Zenodo](https://zenodo.org/record/6334810#.YiYgWHrMJro).
```
Comment (yfukai) [inline README.md:23]: Thanks for adding data in Zenodo! The DOI URL is supposed to be more stable.
https://doi.org/10.5281/zenodo.6334810
Can you change it to this URL?

**Review by yfukai — COMMENTED** [review]
I think the URL should be DOI. The other looks great!

**Review by Nicholas-Schaub — DISMISSED** [review]
Once @yfukai changes about the doi are completed, this Is ready to merge once conflicts are resolved.

**Comment by Nicholas-Schaub** [comment]
@yfukai Can you resolve conflicts?

**Comment by yfukai** [comment]
@Nicholas-Schaub done!

**Comment by yfukai** [comment]
Maybe we can delete Dropbox link?

**Review by yfukai — APPROVED** [review]

**Pull request merged**